### PR TITLE
Fix/v2-submission

### PR DESCRIPTION
**Pull request by bryn-pins**
## Description of change

- Added a missing doc type mapping
- improved logging for submissions
- attempts to send both emails on appeal submission

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/Planning-Inspectorate/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/appeals-service-api/src/services/back-office-v2/index.js [hunk 1]
```diff
@@ -72,25 +73,48 @@ class BackOfficeV2Service {
...
+		try {
+			await sendSubmissionReceivedEmailToLpaV2(appellantSubmission, email);
+		} catch (err) {
+			logger.error({ err }, 'failed to sendSubmissionConfirmationEmailToAppellantV2');
+			emailFailed = true;
+		}
 
-		await sendSubmissionReceivedEmailToLpaV2(appellantSubmission, email);
+		if (emailFailed) {
+			throw new Error('failed to send submission email');
```
Comment (topickering) [inline packages/appeals-service-api/src/services/back-office-v2/index.js:116]: Is there any value in having a different/expanded emailFailed for lpa and appellant in order to make it easier to identify which failed?

File: packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js [hunk 2]
```diff
@@ -39,6 +39,7 @@ const documentTypeMap = {
 	[documentTypes.uploadOriginalApplicationForm.name]: 'originalApplicationForm',
 	[documentTypes.whoWasNotified.name]: 'whoNotified',
 	[documentTypes.conservationMap.name]: 'conservationMap',
+	[documentTypes.representationUpload.name]: 'otherPartyRepresentations',
 	[documentTypes.planningOfficersReportUpload.name]: 'planningOfficerReport'
```
Comment (topickering) [inline packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js:43]: When we add the s78 appeal etc we're going to have to add quite a few doc types, maybe not one for now but at some point we might want to move this to its own file

**Review by topickering — DISMISSED** [review]
Two minor comments but no direct action required, looks good

File: packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js (hunk 2 above)
Comment (bryn-pins) [reply packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js:43]: It should probably just be a property on documentTypes

File: packages/appeals-service-api/src/services/back-office-v2/index.js (hunk 1 above)
Comment (bryn-pins) [reply packages/appeals-service-api/src/services/back-office-v2/index.js:116]: I was going to say we'd have the logger.error but I used the wrong function name

**Review by topickering — APPROVED** [review]

**Pull request merged**